### ISSUE TITLE
On MacOs with Voiceover, one can't know the cell number in the notebook view

### ISSUE THREAD

**Issue by mohammad-tau** [microsoft/vscode#235472]
Type: <b>Bug</b>

	1.	Launch VS Code, open a Jupyter notebook, enable VoiceOver, and focus on the cells view (focused by default).
	2.	Navigate between cells using the up and down arrow keys.
	3.	Observe how VoiceOver announces the cells.

Expected Behavior:
In addition to the existing announcement, VoiceOver should report the cell’s positional information, e.g., “Cell 3 of 20.”

Actual Behavior:
No such announcement is made.

Note:
This appears to be a regression, possibly introduced in VS Code 195 or a similar version. While the removal of the aforementioned announcement is a welcome change on Windows—where screen readers like NVDA automatically provide such information—it creates an issue on macOS, as VoiceOver fails to replicate this behavior.

As a result, users cannot easily determine their position within the notebook, leading to a more confusing experience.

VS Code version: Code 1.95.3 (f1a4fb101478ce6ec82fe9627c43efbf9e98c813, 2024-11-13T14:50:04.152Z)
OS version: Darwin arm64 24.2.0
Modes:
Remote OS version: Linux x64 5.4.0-176-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Max (10 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|3, 3, 4|
|Memory (System)|32.00GB (0.60GB free)|
|Process Argv|--crash-reporter-id 9b5d4c22-c9c6-418d-9268-ef070e8d74f8|
|Screen Reader|yes|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: 10.33.80.124|
|OS|Linux x64 5.4.0-176-generic|
|CPUs|Intel(R) Xeon(R) Silver 4316 CPU @ 2.30GHz (80 x 0)|
|Memory (System)|377.54GB (370.51GB free)|
|VM|0%|
</details><details><summary>Extensions (16)</summary>

Extension|Author (truncated)|Version
---|---|---
jupyter-keymap|ms-|1.1.2
remote-ssh|ms-|0.115.1
remote-ssh-edit|ms-|0.87.0
vscode-remote-extensionpack|ms-|0.26.0
remote-explorer|ms-|0.4.3
remote-server|ms-|1.5.2
debugpy|ms-|2024.12.0
python|ms-|2024.20.0
vscode-pylance|ms-|2024.12.1
jupyter|ms-|2024.10.0
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.21
tensorboard|ms-|2023.10.1002992421
vscode-jupyter-cell-tags|ms-|0.1.9
vscode-jupyter-slideshow|ms-|0.1.6
code-spell-checker|str|4.0.21

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythonnoceb:30805159
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
dvdeprecation:31068756
dwnewjupyter:31046869
2f103344:31071589
nativerepl2:31139839
pythonrstrctxt:31112756
nativeloc1:31192215
cf971741:31144450
iacca1:31171482
notype1:31157159
5fd0e150:31155592
dwcopilot:31170013
stablechunks:31184530

```

</details>

**Comment by amunger** [comment]
@mohammad-tau - Do you get the index announced for other listviews, like the search results (ctrl+shift+f) or the peek view when exploring variable references with shift+F12?

**Comment by mohammad-tau** [comment]
Yes, from what I can observe, such an announcement is available. I hear X of Y, where X is the index in the list starting from 1, and Y is the total number of items in the list.
